ext/dom/namednodemap.c

```diff
@@ -235,34 +226,22 @@ PHP_METHOD(DOMNamedNodeMap, getNamedItemNS)
 
 	if (itemnode) {
 		DOM_RET_OBJ(itemnode, objmap->baseobj);
-	} else {
-		RETVAL_NULL();
 	}
```

Comment:
This could still be useful to communicate that the method either returns a DOM object or `null`, although `RETURN_NULL();` would be "better" than the retval variant

Reply:
All methods return NULL by default unless the return value is overwritten. In DOM we usually don't do the explicit RETURN_NULL(). For me it's just noise.

Reply:
Well if none of the other methods do it, then lets be consistent about it and remove it.